index.css

```diff
@@ -116,8 +115,46 @@ header {
   rotate: -360deg;
 }
 
+nav {
+  display: flex;
+}
+
+#menu-icon {
+  align-self: center;
+  justify-self: flex-end;
+}
+
+#close-menu-icon {
+  width: 14px;
+  height: 14px;
+  align-self: flex-end;
+}
+
 .nav-links {
   display: none;
+  flex-direction: column;
+  justify-content: space-between;
```

Comment:
- [ ] Hi currently you have your nav links in column even on the desktop view which does not follow the Figma design kindly fix it.